mrp_customization/tests/test_split_mo_flow.py

```diff
@@ -0,0 +1,45 @@
+from odoo.tests.common import TransactionCase
+from odoo.exceptions import ValidationError
+
+
+class TestSchoolRefernce(TransactionCase):
+    def setUp(self):
+        """New setup method to create products, bom and mo #T7141"""
+        super(TestSchoolRefernce, self).setUp()
+        self.product_1 = self.env["product.template"].create({"name": "product_1"})
```

Comment:
- Stor env. in variable and use that variable instead of taking the same env. for creating product.product records.